package.json

```
@@ -1,8 +1,9 @@
 {
   "name": "is-mergeable-object",
-  "version": "1.1.0",
+  "version": "1.1.1",
```

```suggestion
  "version": "1.1.0",
```

My deploy script does this for me using `npm version`